tests/src/Unit/AuthManagerTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+use Drupal\social_api\AuthManager\OAuth2Manager;
+use Drupal\social_api\AuthManager\OAuth2ManagerInterface;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Defines OAuth2Manager.
+ *
+ * @Annotation
+ */
+class AuthManagerTest extends UnitTestCase {
+
+  /**
+   * Define __construct function.
+   */
+  public function __construct() {
+    parent::__construct();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+  }
+
+  /**
+   * Tests for class Network.
+   */
+  public function testOAuth2Manager() {
```

Comment:
You're not following my recommendation of separating things into logical steps

Reply:
I have added newlines for the logical steps.